github/resource_github_repository_test.go

```diff
@@ -557,6 +631,33 @@ func testAccCheckGithubRepositoryTemplateRepoAttribute(n string, repo *github.Re
 	}
 }
 
+func testAccCheckGithubVulnerabilityAlerts(n string, expected bool) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		rs, ok := s.RootModule().Resources[n]
```

Comment:
it may be redundant to check w/in the state here as a call to this function in the acceptance tests is preceded by `testAccCheckGithubRepositoryExists` which checks this for us, so the logic here can be omitted similar to`testAccCheckGithubRepositoryAttributes`